cypress/utils/data_utils.ts

```diff
@@ -1,21 +1,29 @@
 import * as faker from "faker";
 
-export function getStakeholderName(): string {
+export function getFullName(): string {
     return faker.name.findName();
 }
 
-export function getStakeholderEmail(): string {
+export function getEmail(): string {
     return faker.internet.email();
 }
 
-export function getBusinessServiceName(): string {
+export function getCompanyName(): string {
     return faker.company.companyName();
 }
 
-export function getBusinessServiceDescription(): string {
+export function getSentence(): string {
     return faker.lorem.sentence();
 }
```

Comment:
what are thoughts about making it common for Description and Comment. Rather having getSentence? 
IMO, Having description/comment in function name will make more readable while creating objects.

@sshveta @nitishSr let me know your views?

Reply:
Yes getDescritpion() is better . It can be used for comments too because comments are at fewer places.

Reply:
Yes this one is tricky, but again in some cases having getComments or description is too generic. May be getDescription can be finalized?